### ISSUE TITLE
Learning rate and loss value for small number of epoch

### ISSUE THREAD

**Issue by fangliu518** [peteryuX/arcface-tf2#21]
Thanks for the project. During your training, did you change the base learning rate to smaller number over training? How many total epoch did you train for your pre trained model? What is the loss value at the end of training? I found my training loss at around 2.1 is reducing very slowly, do you have any suggestions?

**Comment by alexeybozhchenko** [comment]
@fangliu518 Hi. I know its a little late, but maybe you figured out the number of epochs to train?
